logicrunner/builtin/contract/costcenter/costcenter.go

```diff
@@ -35,20 +35,24 @@ func New(commissionWallet insolar.Reference, currentTariff insolar.Reference) (*
 	}, nil
 }
 
+// Set tariffs
```

Comment:
Comment on exported var `SetTariffs ` should be of the form `SetTariffs ...`
Check golang comment style
Check our Go code style, section 7 (Comments)